TakingITGlobal/commit2act
# [Tig-763] - MultiSelectTile Component

# Description

In this PR, I created the MultiSelectTile component. This component will be used in the Sign-Up flow for the action type select tiles. The user can:

- [ ] Can tab focus between each card, selecting (or deselecting) each using Spacebar or Enter
- [ ] Can select or deselect one, multiple, all, or none of the cards
- [ ] Can click either the image or label for each action card to select it

This PR has images which are placeholders and the placeholder images will be changed into the images from the constants folder.

# Testing

1. Pull down and install the branch (`npm install`)
2. Run all tests (`npm run test:basic`) and confirm they pass
3. Run Storybook (`npm run storybook`)
4. Under MultiSelectTile make sure the Unchecked version you CAN see the avatar
5. Under MultiSelectTile make sure the checked version you CANNOT see the avatar (grey checkmark)
6. Under MultiSelectTileGroup you are able to check multiple tiles

# Images
![image](https://github.com/TakingITGlobal/commit2act/assets/132935720/b1b8f157-571e-43cc-bba9-89e7ad1b7dc7)
![image](https://github.com/TakingITGlobal/commit2act/assets/132935720/68158329-7337-420d-b832-58b63e3a8fe9)
![image](https://github.com/TakingITGlobal/commit2act/assets/132935720/c33b3262-d3dd-441a-8734-653e64854d26)
![image](https://github.com/TakingITGlobal/commit2act/assets/132935720/ddb97b40-9df4-4a97-b2fd-d74f6bc890a9)
